### PR TITLE
Polish and edge case fixes on `attack` command

### PR DESCRIPTION
**Pull request by ohboyotero**
* Added verbose description to `attack` command help text, including actual syntax and description of params.
* Gracefully handled no input case.
* Gracefully handled the invalid input case.

### CODE REVIEW

File: commands/combat_commands.py [hunk 1]
```diff
@@ -40,7 +47,15 @@ def func(self):
         location = caller.location
         args = self.args
         arts = caller.db.arts
+
+        if len(args) == 0:
+            return caller.msg("You need to specify a target and art, dumb-dumb. See `help attack` for syntax.")
```
Comment (ohboyotero) [inline commands/combat_commands.py]: Maybe let's use non-offense user-derogatory terminology here? Perhaps call the offending user a "schmoo"

Comment (ohboyotero) [reply commands/combat_commands.py]: Great point, @ohboyotero ! Good feedback. Done. 👍

Comment (ohboyotero) [reply commands/combat_commands.py]: Hey @ohboyotero , great that you took the feedback so well. Sometimes people get touchy on reviews. Thanks for the change!

Comment (ohboyotero) [reply commands/combat_commands.py]: No prob @ohboyotero , happy to help.

Comment (elee-p3) [reply commands/combat_commands.py]: next time, I'll make sure you get some exceptional feedback than from someone other than @ohboyotero

**Pull request merged**